mobify/platform-scaffold
# [Integration Manager] Recover from stale session data

If a tab with the site is left open for long enough, the data in `sessionStorage` will no longer be recognized by the backend. This PR detects the error conditions and retries without the stale data. 

 **JIRA**: N/A (discovered during https://mobify.atlassian.net/browse/WEBDATA-10)

## Changes
- If the auth token refresh fails, start over with a new token
- If the cart is missing, create a new one

## How to test-drive this PR
- Leave a tab open with the Demandware Merlin's long enough for the token to get stale
- Reload and see that there are no auth errors in the console.
